include/mbgl/util/tileset.hpp

```diff
@@ -14,22 +14,27 @@ namespace mbgl {
 class Tileset {
 public:
     enum class Scheme : bool { XYZ, TMS };
+    enum class Encoding : bool { Mapbox, Terrarium };
 
     std::vector<std::string> tiles;
     Range<uint8_t> zoomRange;
     std::string attribution;
     Scheme scheme;
+    Encoding encoding;
```

Comment:
`Tileset` primarily mirrors `TileJSON`, so it would be helpful to add a comment here that `encoding` is not a part of the spec (as yet?).